scriptworker/cot/verify.py

```diff
@@ -1030,11 +1034,61 @@ def verify_link_in_task_graph(chain, decision_link, task_link):
     }
 
 
-def _get_additional_github_releases_jsone_context(parent_link, decision_link):
+async def _get_additional_github_releases_jsone_context(decision_link):
+    context = decision_link.context
+    source_env_prefix = context.config['source_env_prefix']
+    task = decision_link.task
+    repo_url = get_repo(task, source_env_prefix)
+    repo_owner, repo_name = extract_github_repo_owner_and_name(repo_url)
+    tag_name = get_revision(task, source_env_prefix)
```

Comment:
Is there ever a chance that this isn't a tag? If so, should we catch that here or in `GithubRepository`, or will we just fail out in chain of trust verification?